about-peer-review/review-benefits.md

```diff
@@ -0,0 +1,38 @@
+# The Benefits of Submitting a Package to pyOpenSci
+
+
+There is a lot to gain from submitting your open source Python package to pyOpenSci for peer review.
+
+## Supportive feedback from your peers will improve the quality and usability of your package 
+
+Our review process is focused on improving software quality and sustainability. We aim to provide useful feedback to package authors and for our review process to be open and supportive.
+
+## Support from the pyOpenSci community of maintainers and experts
+
+ Once your package is accepted, your package will receive support from pyOpenSci community. You'll retain ownership and control of your package. However, if you need help with ongoing maintenance issues, we will try to find people who can help. You will also have accessed to our Slack community to engage with other maintainers and to seek help. 
+  
+##  pyOpenSci will promote your package online
+
+pyOpenSci will promote your package online through our social media accounts 
+and website catalog. We will also promote blogs and posts that you create about your 
+package. You will also be invited to write a blog post on our website highlighting 
+your package. 
+
+   - [webpage](https://www.pyopensci.org/python-packages/), 
+   - [blog](https://www.pyopensci.org/blog/), and 
+   - [social media account(s)](https://twitter.com/pyopensci).
+
+## Long term maintenance support
+
+ pyOpenSci values maintained open source tools. In the case that you need to step down from maintaining your tool, or in the case that you simply wish to build a larger maintainer team, pyOpenSci will help you find new people to support. In the case that it makes more sense for you to sunset your package, we will support you in that effort as well. 
+
+## Adherence to community-accepted standards
+
+ pyOpenSci is keeping close watch on Python-specific community accepted standards in packaging. Thus our review process will help to enforce standards where we can.
+
+## Reduced duplication of Python tools that support open science
+
+pyOpenSci has a broad view across the scientific Python ecosystem. When you submit your package for review we will ask you to identify other packages in the ecosystem that may have similar functionality. In the long run we hope to identify overlap in package functionality and help maintainers work together on similar tools rather than maintaining overlapping functionality. 
+
+## Your package can be fast tracked and accepted by JOSS
+pyOpenSci packages that are in scope for the [Journal of Open-Source Software](https://joss.theoj.org/) and add the necessary accompanying short `paper.md` file, can, at the discretion of JOSS editors, benefit from a fast-tracked review process. [Read more about our partnership with JOSS here](pyopensci-and-joss).
```

Comment:
> [Read more about our partnership with JOSS here](pyopensci-and-joss).
Is this link missing or should it be different text? It doesn't render when I build locally

Reply:
good catch! there is a funny sphinx "feature" where if a link that is local is broken it just doesn't render as a link 😆 fixed. the file name was wrong as i had updated it.